web/portal/client/src/entities/Domain/SelectOptions.ts

```diff
@@ -0,0 +1,28 @@
+import { DropdownChoices } from '@irontec/ivoz-ui';
+import defaultEntityBehavior from '@irontec/ivoz-ui/entities/DefaultEntityBehavior';
+import { SelectOptionsType } from '@irontec/ivoz-ui/entities/EntityInterface';
+import store from 'store';
+
+const DdiSelectOptions: SelectOptionsType = ({
```

Comment:
DdiSelectOptions (and other Ddi mentions in this file) looks like a copy pasterino issue